pyat/at/lattice/elements.py

```diff
@@ -490,6 +490,66 @@ def stds(self):
     @property
     def means(self):
         return self._means
+        
+        
+class SliceMoments(Element):
+    """Element to compute bunches mean and std"""
+
+    def __init__(self, family_name: str, nslice: int, **kwargs):
```

Comment:
having nslice as position argument doesn't work nicely as described in my previous comment

Reply:
See comment below, I cannot reproduce this problem.